examples/http-disable-chunking/http_disable_chunking.bal

```diff
@@ -4,49 +4,46 @@ import ballerina/log;
 //The HTTP client's chunking behaviour can be configured as auto, always, or never.
 //In this example, it is set to as never, which means that chunking never happens irrespective of how it is specified
 //in the request. When chunking is set to auto, chunking is done as specified in the request.
-endpoint http:Client clientEndpoint {
```

Comment:
It's better if we could do the client initialization here. Because the above comment describes the client